odl/operator/default_ops.py

```diff
@@ -913,9 +812,9 @@ def __str__(self):
 
 class ResidualOperator(Operator):
 
-    """Operator that calculates the residual ``op(x) - vec``.
+    """Operator that calculates the residual ``op(x) - y``.
 
-    ``ResidualOperator(op, vector)(x) <==> op(x) - vec``
+    ``ResidualOperator(op, y)(x) <==> (x --> op(x) - y)``
```

Comment:
dislike this syntax change. We used the old one quite extensively and I find it more clear since the two sides are explicit.

Here, the left hand side is explicit, but the right hand side looks like a function?